src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -28,6 +32,47 @@ import {
 
 import featureFlags from '../featureFlags';
 
+const AdditionalInfos = () => (
```

Comment:
To clean up the code in the main `render` I moved this JSX into its own local component